bin/test-cleanup

```diff
@@ -80,13 +85,13 @@ if [[ ${namespaces_controlplane[*]}   || \
 fi
 
 echo "cleaning up viz extension resources, if present [${k8s_context}]"
-linkerd viz uninstall 2> /dev/null | kubectl --context="$k8s_context" delete -f -
+"$linkerd_path" viz uninstall 2> /dev/null | kubectl --context="$k8s_context" delete -f -
 
 echo "cleaning up multicluster resources, if present [${k8s_context}]"
-linkerd mc uninstall 2> /dev/null | kubectl --context="$k8s_context" delete -f -
+"$linkerd_path" mc uninstall 2> /dev/null | kubectl --context="$k8s_context" delete -f -
 
 echo "cleaning up jaeger extension resources, if present [${k8s_context}]"
-linkerd jaeger uninstall 2> /dev/null | kubectl --context="$k8s_context" delete -f -
+"$linkerd_path" jaeger uninstall 2> /dev/null | kubectl --context="$k8s_context" delete -f -
```

Comment:
`linkerd jaeger uninstall` doesn't seem to have that check, but it'd be better to place this before as well.

Reply:
same for viz?

Reply:
Yea I'll just move all the extension uninstalls above the control plane uninstall!